base_layer/wallet/src/transaction_service/protocols/transaction_send_protocol.rs

```diff
@@ -241,16 +241,70 @@ where TBackend: TransactionBackend + Clone + 'static
                 e
             });
 
+        // TODO Actually monitor the send status of this message
         self.resources
```

Comment:
It might add to privacy to only do the propagating call below. Won't the distance calculations mean that the destination node will receive the propagated message directly as well if there it is connected directly?

Reply:
My understanding of the propagation is that it gets sent to the N "closest" peers in your peer list and there is no guarantee that this peer is one of those.

Reply:
I thought it was "kinda" close to the node id, but the source for that info is a few minutes skimming a PR from a few days ago, so can't really comment